MP-APS/Timer.h

```diff
@@ -1,17 +1,32 @@
 #pragma once
-#include <cstdint>
+
+#include <functional>
 
 class Timer {
+	using TimerCallbackFunc = std::function<void()>;
 public:
-	Timer() noexcept;
+	
+	Timer(const double secondsBetweenCallbackTriggers, TimerCallbackFunc callback) 
```

Comment:
`TimerCallbackFunc` is passed by value for performance reasons since we're storing it https://stackoverflow.com/a/18366273/2231969